cloudsim-plus/src/main/java/org/cloudbus/cloudsim/allocationpolicies/power/PowerVmAllocationPolicyMigrationAbstract.java

```diff
@@ -46,8 +46,8 @@
     implements PowerVmAllocationPolicyMigration {
 
     /**@see #getUnderUtilizationThreshold() */
-    private double underUtilizationThreshold = 0.35;
-
+    private double underUtilizationThreshold = 0.3;
```

Comment:
You shouldn't change default values in core classes. If you want a different value, call the setUnderUtilizationThreshold passing the value you want.
The  PowerVmAllocationPolicyMigration didn't provide this setter but now it provides.